config/initializers/setup_mail.rb

```diff
@@ -1,10 +1,11 @@
-ActionMailer::Base.delivery_method = :smtp
-ActionMailer::Base.smtp_settings = {
-  address:        'smtp.sendgrid.net',
-  port:           '587',
-  authentication: :plain,
-  user_name:      ENV['SENDGRID_USERNAME'],
-  password:       ENV['SENDGRID_PASSWORD'],
-  domain:         'heroku.com',
-  enable_starttls_auto: true
+ActionMailer::Base.delivery_method = :smtp	
+ActionMailer::Base.perform_deliveries = true	
+
+ActionMailer::Base.smtp_settings = {	
+  address:        'email-smtp.us-east-1.amazonaws.com',
+  port:           '587',	
+  authentication: :plain,	
+  user_name:      ENV['SMTP_USERNAME'],	
+  password:       ENV['SMTP_PASSWORD'],	
+  domain:         'mylibrarynyc.org',	
```

Comment:
This block seems to conflict with production.rb?

Reply:
Keeping this block. And removing the block in the production.rb file.